src/OpenLoco/src/Environment.cpp

```diff
@@ -56,6 +56,7 @@ namespace OpenLoco::Environment
             "C:/Program Files (x86)/Atari/Locomotion",
             "C:/GOG Games/Chris Sawyer's Locomotion",
             "C:/GOG Games/Locomotion",
+            "C:/Program Files (x86)/Steam/steamapps/common/Locomotion",
```

Comment:
Is Steam still exclusively installed in `Program Files (x86)` nowadays? If not, it might be worth including `Program Files` as well. (cc @ZehMatt, do you know?)

Reply:
It is not, additionally Steam can have more than one library path, the game can be anywhere. I guess its a good default to have but I'm also not a fan of hardcoding paths.

Reply:
Okay, then I suggest we support the other default path as well, at least:
```suggestion
            "C:/Program Files/Steam/steamapps/common/Locomotion",
            "C:/Program Files (x86)/Steam/steamapps/common/Locomotion",
```